src/clj/nnts2/db/note.clj

```diff
@@ -20,3 +20,22 @@
                                    sql/format)
                      {:identifiers utils/snake->kebab})
          first))))
+
+
+(defn get-honeysql-filter-params
+  "modified input params to honeysql format where parameters"
+  [params]
+  (for [[k v] params] [:= k v]))
```

Comment:
How would this handle a check with a an operand that's not a `=`? I suggest not having this and writing different functions for each conditional for the sake of being explicit about what each of them do.

Reply:
will add other operators in the newer prs. Have to solve out they will be parsed at compojure level first.